mne/report.py

```diff
@@ -1556,6 +1562,48 @@ def _render_forward(self, fwd_fname):
                                         repr=repr_fwd)
         return html
 
+    def _render_sensitivity_map(self, fwd_fname, subject, subjects_dir):
+        """Render Sensitivity plots.
+        """
+        caption = u'Sensitivity : %s' % fwd_fname
+        div_klass = 'sensitivity'
+        img_klass = 'Sensitivity'
+        show = True
+
+        fwd = read_forward_solution(fwd_fname, surf_ori=True)
+        mag_map = sensitivity_map(fwd, ch_type='grad', mode='fixed')
+        brain = mag_map.plot(time_label='Gradiometer sensitivity', subjects_dir=subjects_dir, clim=dict(lims=[0, 50, 100]))
+
+        html = []
+        from mayavi import mlab
+        img = mlab.gcf()
+        global_id = self._get_id()
+        img = _fig_to_img(fig=img)
+        html.append(image_template.substitute(img=img, id=global_id,
+                                         image_format='png',
+                                         div_klass=div_klass,
+                                         img_klass=img_klass,
+                                         caption=caption,
+                                         show=True))
+
+        import matplotlib.pyplot as plt
+        fig = plt.figure()
```

Comment:
MPLBACKEND=“Qt4Agg” (sets matplotlib backend) had to be used to avoid TclError on OS X. Should I mention it somewhere in documentation?